### ISSUE TITLE
There is no any NativeModules of Viro in 2.21.1.

### ISSUE THREAD

**Issue by CreadDiscans** [ViroCommunity/viro#53]
# Requirements: 
Please go through this checklist before opening a new issue
  - [x] Review the [documentation](https://docs.viromedia.com/)
  - [x] Search for existing issues in: [viromedia/viro](https://github.com/viromedia/viro/issues) & [ViroCommunity/viro](https://github.com/ViroCommunity/viro/issues)
  - [x] Use the latest [ViroReact release](https://docs.viromedia.com/docs/releases)

## Environment
Please provide the following information about your environment:
1. Development OS: Mac
2. Device OS & Version: iOS 14.4.2
3. Version: ViroReact version and React Native version : ViroReact : 2.21.1, ReactNative : 0.63.4
4. Device(s): iPhone 11

## Description
There is no any NativeModules of Viro.
```
import {NativeModules} from 'react-native'
console.log(NativeModules.VRTARTrackingTargetsModule) //  => null
```

2.21.0 is works. But there is no any modules in only 2.21.1 version

**Comment by mariansam** [comment]
Might be related to #52 and #49. The code prints this when ran on Android:
```js
{"createTargets": [Function nonPromiseMethodWrapper], "deleteTarget": [Function nonPromiseMethodWrapper], "getConstants": [Function anonymous]}
```

**Comment by CreadDiscans** [comment]
It works on android. but on iOS, the code prints null.

**Comment by doranteseduardo** [comment]
Closed in favour of #56
